lnrpc/walletrpc/walletkit.proto

```diff
@@ -734,6 +741,9 @@ message PublishResponse {
     string publish_error = 1;
 }
 
+message RemoveTransactionResponse {
```

Comment:
Users are often confused by an empty RPC message rendered as JSON on the command line... Should we add a simple `status` string return value that can contain a short message? See https://github.com/lightningnetwork/lnd/pull/7762 for what I mean exactly.

Reply:
what about a mix meaning that we only return the status but the final response to the user is a composite of the status and the txid :

```
lncli wallet removetx 2f68ebb33605de1b9d9a15b82d9bae9b26cdca3f8f1c9aa1acab166527de1444
{
	"status": "Successfully removed transaction",
	"txid": "2f68ebb33605de1b9d9a15b82d9bae9b26cdca3f8f1c9aa1acab166527de1444"
}
```

Reply:
That is even better, thanks!